docs/ReleaseNotes.md

```diff
@@ -1,6 +1,66 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 2.1
+Acuminator 2.1 includes the diagnostics, code fixes, enhancements, and bug fixes described in this section, as well as the features that have been implemented in previous versions.
+
+### New Diagnostic and Code Fixes
+In this version, the diagnostic and code fixes have been added for the following issue.
+
+| Code                            | Issue Description                                                | Type    | Diagnostics | Code Fix    | 
+| ------------------------------- | ---------------------------------------------------------------- | ------- | ----------- | ----------- |
+| [PX1007](diagnostics/PX1007.md) | A public class should have a description in the `summary` XML tag. | Warning (Level 3: Informational)	 | Available | Available   |
+
+### Ability to Suppress a Diagnostic by Adding a Code Comment
+In Acuminator 2.1, you can suppress any diagnostic by adding a specific comment directly in code. To add a suppression comment, you select **Suppress the PX10YY diagnostic with Acuminator > in a comment** (where PX00YY is the code of the diagnostic) as a code fix for the diagnostic, as shown in the following screenshot.
```

Comment:
Why the "PX10**"? It can be any Acuminator diagnostic ID.
```suggestion
In Acuminator 2.1, you can suppress any diagnostic by adding a specific comment directly in code. To add a suppression comment, you select **Suppress the PX**** diagnostic with Acuminator > in a comment** (where PX**** is the code of the diagnostic) as a code fix for the diagnostic, as shown in the following screenshot.
```

Reply:
Do we have any others except PX10**?

Reply:
Not for now, but if we add more diagnostic we are going to have more than a hundred.  Imho, it's better to account for that now than rewrite docs later

Reply:
Changed to PXYYYY

Reply:
Actually, we will not rewrite release notes even if we change something)